assets/js/app.js

```diff
@@ -108,6 +108,29 @@ $(function() {
 });
 
 
+// download page os buttons
+
+$(function() {
+  const windowsContent = "Fork the O3DE source and build the engine and tools directly from any branch. For more details, read the documentation.";
+  const macOsContent = "Coming Spring 2022.";
+  $(".download-os-button").click(function(event){
+    event.preventDefault()
+    console.log("OS SELECTED: ", $(this).text());
+    $(".download-os").removeClass("active");
+    $(this).parents().addClass("active");
```

Comment:
This will actually add the "active" class to all parents recursively. I'm not sure it's the intended behavior.

![image](https://user-images.githubusercontent.com/82231674/126009140-8c227172-4b26-42f0-af1a-8bcb55b983fc.png)